File: CS/12_LinksAndActions/RemoveOpenAction.cs
using Spire.Pdf;
using Spire.Pdf.Actions;
using Spire.Pdf.Annotations;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace RemoveOpenAction
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument document = new PdfDocument();

            //Load an existing pdf from disk
            document.LoadFromFile(@"..\..\..\..\..\..\Data\OpenAction.pdf");

            //Remove action
            document.AfterOpenAction = null;

            String result = "Output.pdf";
            //Save the document
            document.SaveToFile(result);

            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/12_LinksAndActions/RemoveOpenAction.vb
Imports Spire.Pdf

Namespace RemoveOpenAction
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object
			Dim document As New PdfDocument()

			' Load a PDF file from a specified path
			document.LoadFromFile("..\..\..\..\..\..\Data\OpenAction.pdf")

			' Set the AfterOpenAction property to null (removes any predefined action)
			document.AfterOpenAction = Nothing

			' Specify the output file name as "result"
			Dim result As String = "result.pdf"

			' Save the modified document to a PDF file
			document.SaveToFile(result)

			' Close the PDF document
			document.Close()

			' Launch the Pdf file
			PDFDocumentViewer(result)
		End Sub

		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace